jina/flow/__init__.py

```diff
@@ -800,11 +798,12 @@ def flow_visualization(self):
 
         mermaid_graph = list()
         i = 0
+
         for k in self._pod_nodes.items():
```

Comment:
better to have then:
`for k, v in self._pod_nodes.items():
     for need in `v.needs`: